### PR TITLE
Add check "tickets must be for same event in order" (#180)

### PR DESCRIPTION
**Pull request by DeKaN**

### CODE REVIEW

File: konfera/tests/test_models.py [hunk 1]
```diff
@@ -135,21 +133,18 @@ def test_left_to_pay2(self):
...
 
 class ReceiptTest(TestCase):
-
     def test_string_representation(self):
         entry = models.Receipt(title="Test Receipt title")
         self.assertEqual(str(entry), entry.title)
 
 
 class RoomModelTest(TestCase):
-
```
Comment (ZoltanOnody) [inline konfera/tests/test_models.py]: Those blank lines after `class` definition where there for readability purpose. I think we use it that way everywhere so let's keep them here as well (for consistency).

File: konfera/tests/test_models.py [hunk 2]
```diff
@@ -276,6 +268,25 @@ def test_clean_and_save_ticket_discount_code(self):
                                                  discount_code=self.discount_code_2)
         self.assertRaises(ValidationError, ticket_with_invalid_code.save)
 
+    def test_save_tickets_for_different_event(self):
+        title1 = random_string(128)
```
Comment (ZoltanOnody) [inline konfera/tests/test_models.py]: Personally I prefer when the title is static, the assertion messages are then more readable and it takes less time to run the tests.

**Comment by DeKaN** [comment]
@ZoltanOnody Fixed it in new commit

**Comment by ZoltanOnody** [comment]
Strange, I am still getting 500 error.

image

Does the code work for you?

Also change " ... for same event" to "... for **the** same event"

**Comment by DeKaN** [comment]
@ZoltanOnody I've added validation for admin form.

File: konfera/admin.py [hunk 3]
```diff
@@ -155,11 +156,23 @@ class LocationAdmin(admin.ModelAdmin):
 admin.site.register(Location, LocationAdmin)
 
 
+class OrderedTicketsInlineFormSet(forms.models.BaseInlineFormSet):
```
Comment (ZoltanOnody) [inline konfera/admin.py]: Create a `forms.py` file and move this code there.

**Comment by DeKaN** [comment]
@ZoltanOnody You press Save button in order form at admin area and:

1. admin form for `Order` calls `full_clean`
1. admin form for `Receipt` calls `full_clean` (**Receipt.order is None**)
1. admin form for each `Ticket` calls `full_clean` (**Ticket.order is None**)
1. all clean passed and form call `save `, but `Ticket.save` (**Ticket.order is not None** at this moment) calls `clean` without catching `ValidationError`

**Comment by ZoltanOnody** [comment]
Thank you!

**Pull request merged**